src/main/java/spongebob/task/Event.java

```diff
@@ -0,0 +1,80 @@
+package spongebob.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import spongebob.exception.SpongebobException;
+
+
+/**
+ * An Event task, contains a dateTime for start and enddate
+ */
+public class Event extends Task {
+
+    protected LocalDate from;
+    protected LocalDate to;
```

Comment:
I think "from"  and "to" may be slightly ambiguous though I'm aware these are the start and end dates. Perhaps "startDate" and "endDate" might be a slightly more suitable variable name as it is a noun?